.github/workflows/dep.yml

```diff
@@ -23,8 +23,11 @@ on:
       - master
       - branch-*
     paths:
-      # dependency check happens only pom changes
+      # when pom changes
       - '**/pom.xml'
+      # when dependency workflow changes
+      - .github/workflows/dep.yaml
```

Comment:
unnecessary changes?

Reply:
To verify using mvnd in dependency workflows. Changes in `dep.yaml` should trigger the jobs defined from the file itself.